doc/gallery-src/analysis/run_hexBlockToRZConversion.py

```diff
@@ -0,0 +1,41 @@
+"""
+Hex block to RZ geometry conversion
+===================================
+Often, parts of a reactor model must be transformed to a different geometry in order to
+perform a certain type of physics calculation. For example, in some fast reactor lattice
+physics calculations, detailed descriptions of control assemblies must be mapped to
+equivalent 1-D cylindrical models.
+
+This example shows how a control assembly defined in full hex-pin detail can be
+automatically converted to an equivalent 1-D RZ case, including an outer ring of fuel to
+drive the case.
+
+This conversion includes rings for control material, gap, cladding (on both sides of each
+ring of control material), coolant, duct, and fuel. The color of the plot is proportional
+to the mass density.
+
+Given this transformation, a 1-D lattice physics solver can be executed to compute
+accurate cross sections.
+
+By automating these kinds of geometry conversions, ARMI allows core designers to maintain
+the design in real geometry while still performing appropriate approximations for
+efficient analysis.
+
+"""
+
+from armi.reactor.tests import test_reactors
+from armi.reactor.flags import Flags
+from armi.reactor.converters import blockConverters
+import armi
+
+armi.configure()
+
+o, r = test_reactors.loadTestReactor()
+
+bFuel = o.r.core.getBlocks(Flags.FUEL)[0]
+bControl = o.r.core.getBlocks(Flags.CONTROL)[0]
+converter = blockConverters.HexComponentsToCylConverter(
+    bControl, bFuel, numExternalRings=1
+)
```

Comment:
This is also pretty Stuff™. What is it doing? What are its arguments? What assumptions does it make about the internal geometry of the Block? What does `numExternalBlocks` mean? Which block is being converted, and why do I need to pass two blocks?

Reply:
Added the keyword arguments and a hyperlink to the api docs at the top.